mobile/src/bundles/common/components/button/styles.ts

```diff
@@ -1,13 +1,63 @@
 import { StyleSheet } from 'react-native';
 
+import { Color } from '../../enums/enums';
+
 const styles = StyleSheet.create({
```

Comment:
Remove some global styles and add it in component directly

Reply:
fix